src/gui/tray/TalkReply.qml

```diff
@@ -0,0 +1,84 @@
+import QtQuick 2.15
+import Style 1.0
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.2
+import com.nextcloud.desktopclient 1.0
+
+ColumnLayout {
+    id: talkLayout
+    spacing: 0
+    
+    Layout.fillWidth: true        
```

Comment:
@camilasan These attached `Layout` properties must be set at the place of usage of `TalkReply`, like:
```
TalkReply {
    Layout.fillWidth: trueLayout.fillHeight: true        
    Layout.minimumWidth: 170
    Layout.minimumHeight: parent.height
}
```